test/javascripts/acceptance/brightcove-modal-test.js

```diff
@@ -0,0 +1,16 @@
+import { click, visit } from "@ember/test-helpers";
+import { acceptance } from "discourse/tests/helpers/qunit-helpers";
+import { test } from "qunit";
+
+acceptance("Discourse Brightcove | upload modal", function (needs) {
+  needs.user();
+  needs.settings({
+    brightcove_enabled: true,
+  });
+
+  test("can display modal", async (assert) => {
```

Comment:
just for consistency with all tests

```suggestion
  test("can display modal", async function (assert) {
```